circle.yml

```diff
@@ -1,6 +1,8 @@
 machine:
   environment:
     PATH: "${PATH}:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
+  node:
+    version: 6.11.1 # current is 8.2.1
```

Comment:
This comment feel superfluous, and will become stale quickly.  We try to standardise on the LTS version, which is `6.x` until October.